view/frontend/web/js/view/catalog/product/product-protection-offer.js

```diff
@@ -153,6 +156,7 @@ define(['jquery', 'cartUtils', 'extendSdk', 'ExtendMagento'], function (
                     quantity,
                   }).then(cartUtils.refreshMiniCart)
                 }
+                $(eventObj.target.parentElement).trigger('submit')
```

Comment:
If `eventObj` is null, then what would happen? Would it break UI in someway? Can you test that?